CHANGELOG.rst

```diff
@@ -7,7 +7,26 @@ This document describes changes between each past release.
 1.2 (unreleased)
 ----------------
 
-- Nothing changed yet.
+**Breaking changes**
```

Comment:
With https://github.com/mozilla-services/cliquet/pull/96 : 

* rename of cache_url cache_backend
* session table in postgresql can be dropped after upgrade